### ISSUE TITLE
The latest patch has not been updated in PyPi

### ISSUE THREAD

**Issue by shasankp000** [RapidAI/RapidLaTeXOCR#29]
So I was trying to get the ocr data from a math screenshot using Rapid latex ocr when I encountered the very peculiar `ValueError`

`ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()`

Following issue #27 , I see that the commit was merged into the repo, but it hasn't been updated in PyPi, so I tried 

`pip3 install git+https://github.com/RapidAI/RapidLaTeXOCR.git`

Which results in this error

```
Defaulting to user installation because normal site-packages is not writeable
Collecting git+https://github.com/RapidAI/RapidLaTeXOCR.git
  Cloning https://github.com/RapidAI/RapidLaTeXOCR.git to /tmp/pip-req-build-p1j_fsrz
  Running command git clone --filter=blob:none --quiet https://github.com/RapidAI/RapidLaTeXOCR.git /tmp/pip-req-build-p1j_fsrz
  Resolved https://github.com/RapidAI/RapidLaTeXOCR.git to commit da454813114b37fea7bedc946d45bec5b3940e40
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [6 lines of output]
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/tmp/pip-req-build-p1j_fsrz/setup.py", line 9, in <module>
          from get_pypi_latest_version import GetPyPiLatestVersion
      ModuleNotFoundError: No module named 'get_pypi_latest_version'
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details
```

Please fix the bug and update the repository in PyPi

**Comment by SWHL** [comment]
Thanks for your renminder. I had released the v0.0.8, and you can try it again.

**Comment by shasankp000** [comment]
Okay. Thank you for making this awesome repo!
